runtime/sam/expr/function/time.go

```diff
@@ -46,3 +49,49 @@ func (b *Bucket) Call(ectx expr.Context, args []zed.Value) zed.Value {
 	}
 	return zed.NewTime(nano.Ts(v).Trunc(bin))
 }
+
+var strftimeSet strftime.SpecificationSet
+
+func init() {
+	strftimeSet = strftime.NewSpecificationSet()
+	strftimeSet.Delete('n') // newline
+	strftimeSet.Delete('R') // equivalent to %H:%M
+	strftimeSet.Delete('r') // equivalent to %I:%M:%S %p
+	strftimeSet.Delete('t') // tab
+	strftimeSet.Delete('T') // equivalent to %H:%M:%S
+	strftimeSet.Delete('v') // equivalent to %e-%b-%Y
+	strftimeSet.Delete('V') // the week number of the year (Monday as the first day of the week) as a decimal number (01-53)
+	strftimeSet.Delete('z') // timezone
+	strftimeSet.Delete('Z') // timezone
```

Comment:
I think we should keep these since they are all in the POSIX strftime specification.

Reply:
ok

Reply:
I was weary about allowing timezone since we don't really support timezones with zed time, but I suppose we can allow timezones and it will always be `GMT`.